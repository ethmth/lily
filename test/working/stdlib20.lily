


let a::int = [5, 1, 2, 4, 5, 6]

let b::int = [7, 6, 8, 2, 1,1]

let c::int = shuffle(a,b)

print(c)