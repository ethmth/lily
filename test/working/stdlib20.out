[5, 1, 2, 6, 5, 6, 7, 4, 8, 2, 1, 1]
